operator/cmd/actions/start.go

```diff
@@ -34,6 +39,42 @@ func operatorMain(ctx *cli.Context) error {
 		return err
 	}
 
+	// hash version
+	hash := sha3.NewLegacyKeccak256()
+	hash.Write([]byte(ctx.App.Version))
+
+	// get hash
+	version := hash.Sum(nil)
+
+	// sign version
+	signature, err := crypto.Sign(version[:], operatorConfig.EcdsaConfig.PrivateKey)
+	if err != nil {
+		return err
+	}
+
+	body := map[string]interface{}{
+		"version":   ctx.App.Version,
+		"signature": signature,
+	}
+	bodyBuffer := new(bytes.Buffer)
+
+	bodyReader := json.NewEncoder(bodyBuffer)
+	err = bodyReader.Encode(body)
+	if err != nil {
+		return err
+	}
+
+	// send version to metrics server
```

Comment:
small nit but this comment confused me 😛. It should be the operator tracker server